sageodata-apispec.yml

```diff
@@ -2530,20 +2607,86 @@ components:
           type: string
         groundElevation:
           format: int32
+          description: The well's current ground elevation in metres.
           type: integer
         referenceElevation:
           format: double
+          description: The well's current elevation from the reference point in metres.
           type: number
         referencePointType:
+          description: Name of the reference point, e.g. TOC (Top of Casing)
+          enum:
+          - GRD
+          - KB
+          - CAB
+          - TOC
+          - FLR
+          - PBP
+          - REF
+          - STP
+          - SCB
+          - WEL
+          - WLC
+          - WIN
+          - WDC
+          - COL
+          - FLN
           type: string
         surveyMethod:
+          description: The survey method used, e.g. GPSSN (GPS Standalone Navigational)
+          enum:
+          - MAP
+          - GPS
+          - SUR
+          - PRO
+          - UKN
+          - CAL
+          - DIG
+          - GCN
+          - SAN
+          - INFER
+          - AMGRD
+          - CLGRD
+          - UCMAP
+          - IMGD
+          - PHOTO
+          - VERBL
+          - GPSUN
+          - GPSSN
+          - GPSAP
+          - GPSRD
+          - GPSPD
+          - GPSSL
+          - DOCM
+          - TRX
+          - GPSDS
+          - GPSDM
+          - GPSDG
+          - COCH
+          - COOD
+          - EDM
+          - LOCAL
+          - ODOM
+          - GISP
+          - GISW
+          - GPSSK
+          - GPSSS
+          - GTRAV
+          - GOEAR
+          - GOMAP
+          - DEM
+          - GPSAU
```

Comment:
Same as previous comment.. it will be good to provide a description of what these codes mean. 
If you know what SA Geodata table this is from, I believe the description would be in there.

Reply:
There is a definition for surveyMethod in _wcr-coordinatedata_. You can use the definition ffrom there. 
It is using the same lookup table (under the hood) in SA Geodata.